Sources/UIKit/EmbeddedTransformation.swift

```diff
@@ -60,7 +60,7 @@ internal enum EmbeddedTransformationHelpers {
         let representations = styleAttributes[BonMotTransformationsAttributeName] as? [StyleAttributes] ?? []
         let results: [T?] = representations.map { representation in
             for type in embeddedTransformationTypes {
-                if let transformation = type.from(dictionary: representation) as? T {
+                if let transformation = type.init(dictionary: representation) as? T {
```

Comment:
for line 61, we can change this to a compactMap so we don't need to go through the collection again, and with flattop
```suggestion
         let results: [T] = representations.compactMap { representation in
```